CloudWatch: remove references to json in template variable docs (#48593)
grafana/grafana: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md

## Query variable

> The CloudWatch data source provides the following queries that you can specify in the `Query Type` field in the Variable edit view. They enable you to fill a variable's options list with values such as `regions`, `namespaces`, `metric names` and `dimension keys/values`.

Reviewer: ```suggestion
You can use the following CloudWatch data source queries to specify the `Query Type` field in the Variable edit view. Use them to fill a variables options list with values like `regions`, `namespaces`, `metric names`, and `dimension keys/values`.
```

Variables can be used in the variable form. Refer to the [variable syntax documentation]({{< relref "../../variables/syntax.md" >}}).

## ec2_instance_attribute examples

### Filters

> The `ec2_instance_attribute` query takes in `filters`, each as a filter name and a comma-separated list of values.

Reviewer: ```suggestion
The `ec2_instance_attribute` query takes in `filters`as a filter name and a comma-separated list of values.
```

```javascript
{"Environment":${env:json}}
```

## ec2_instance_attribute examples

### JSON filters
### Filters

The `ec2_instance_attribute` query takes `filters` in JSON format.
The `ec2_instance_attribute` query takes in `filters` as a filter name and a comma-separated list of values.
You can specify [pre-defined filters of ec2:DescribeInstances](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html).

> Note that the actual filtering takes place on Amazon's servers, not in Grafana.

Reviewer 2: Is this line still relevant or was it only specific to the JSON format?

Author: Still relevant! Those are the filter names on aws's end.
